ionic_user_interface/src/views/UploadRoute.vue

```diff
@@ -0,0 +1,313 @@
+<template>
+  <ion-page>
+    <ion-content :fullscreen="true" ref="content">
+      <ion-grid>
+        <ion-row color="primary" class="ion-align-items-center ion-justify-content-center">
+          <ion-col class="ion-align-self-center" size-md="6" size-lg="5" size-xs="12">
+            <div class="ion-text-center">
+              <h1>Upload Route</h1>
+            </div>
+            <div class="ion-padding ion-text-center">
+              <MessageBox ref="msgBox" :color="msgBoxColor" class="rounded margin" />
+              <form @submit="onSubmit">
+                <ion-item class="rounded margin">
+                  <ion-label position="floating">Route name</ion-label>
+                  <ion-input
+                    v-model="routeNameText"
+                    inputmode="text"
+                    type="text"
+                    maxlength="30"
+                    autofocus
+                    required
+                  />
+                  <!-- Fix to prevent enter submit -->
+                  <ion-input class="ion-hide"></ion-input>
+                </ion-item>
+                <ion-list class="rounded margin">
+                  <ion-item>
+                    <ion-label>Grade</ion-label>
+                  </ion-item>
```

Comment:
I think we can remove this because -

#### This form looks a little confusing (As if there is a field besides "Grade")
![image](https://user-images.githubusercontent.com/19868916/122858767-68da1f80-d34d-11eb-9e4f-3dd892290dcc.png)

<hr/>

#### Compared to this
![image](https://user-images.githubusercontent.com/19868916/122858744-5a8c0380-d34d-11eb-9a6c-17eb4bd3664c.png)

<br/>

But maybe you have better ideas?

Reply:
Putting grades may be better but maybe not that position

Reply:
Or maybe we can put different font for the grade

Reply:
I left it like that cuz I had no better ideas, but yeah I guess we can just remove the label